server/channels/app/import_functions.go

```diff
@@ -1233,6 +1290,9 @@ func (a *App) importReplies(rctx request.CTX, data []imports.ReplyImportData, po
 		if replyData.EditAt != nil {
 			reply.EditAt = *replyData.EditAt
 		}
+		if replyData.IsPinned != nil {
+			reply.IsPinned = *replyData.IsPinned
```

Comment:
It doesn't seem like we added a test for this. Would it be doable?

Reply:
Added a new test case for this one: https://github.com/mattermost/mattermost/pull/28286/commits/45b75624b31bf3035cbb08bbf6258ad30dba7c12